tools/tentacle_manager/tentacle_package_manager.py

```diff
@@ -69,19 +69,16 @@ def _process_action_on_module(self, action, module_type, module_subtype,
                     did_something = False
 
             # Update local __init__
-            line_in_init = "from .{0} import *\n".format(module_name)
-            init_file = "{0}/{1}/{2}/{3}/{4}".format(TENTACLES_PATH,
-                                                     TENTACLE_TYPES[module_type],
-                                                     TENTACLE_TYPES[module_subtype],
-                                                     target_folder,
-                                                     PYTHON_INIT_FILE)
+            line_in_init = f"from .{module_name} import *\n"
+            init_file = f"{TENTACLES_PATH}/{TENTACLE_TYPES[module_type]}/{TENTACLE_TYPES[module_subtype]}/" \
```

Comment:
I think that a [_Multi-line string syntax_](https://www.smallsurething.com/multi-line-strings-in-python/) will be better here. (with `"""` )